grakn-test/src/test/java/ai/grakn/test/engine/tasks/BackgroundTaskTestUtils.java

```diff
@@ -90,15 +95,15 @@ private static void waitForStatus(TaskStateStorage storage, TaskState task, Set<
     public static synchronized void waitForStatus(TaskStateStorage storage, TaskId task, Set<TaskStatus> status) {
         Instant initial = Instant.now();
         long duration = Duration.between(initial, Instant.now()).toMillis();
-        while(duration < 30000) {
+        while(duration < 120000) {
```

Comment:
I should probably log the final duration so we can see what the distribution looks like. This is just to avoid the occasional annoying test failure